api/client/samples/analogous_years/readme.md

```diff
@@ -0,0 +1,164 @@
+# Analogous Years
```

Comment:
let's remove redundant info between readme and notebook now that the latter exists
readme can jus tbe a short cmd line example
and then can point to notebook for more complex usage

Reply:
There is still some overlap. However, I have removed much of it.